node/build.sh

```diff
@@ -0,0 +1,53 @@
+NODE_VERSION=2.3.7
```

Comment:
Please run your script through [shellcheck](https://www.shellcheck.net/).
Also follow the example from [nginx](https://github.com/unikraft/static-pie-apps/blob/master/nginx/README.md).